starfish/image/_filter/zero_by_channel_magnitude.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, thresh: int, normalize: bool) -> None:
         """
         self.thresh = thresh
         self.normalize = normalize
+        self.is_volume = is_volume
```

Comment:
doesn't look like anyone uses this...?

Reply:
@shanaxel42 : I wasn't sure either which is why I passed the buck. 😉

Reply:
My intuition is that we should not include this as it implies it does something it does not.  I think there are filters that do not accept this format.

Second choice: raise ValueError if is_volume is True.

Reply:
Updated to not include the flag here